From e299076fefd9649f78f853865d4745043e50813c Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Tue, 20 Sep 2016 22:25:37 +0200
Subject: [PATCH] Fix old-style function definition

* mach/errstring.c (mach_error_type, mach_error_string): Fix old-style
function definition.
---
diff --git a/mach/errstring.c b/mach/errstring.c
index 52cf1eb46e..c00b02c2a6 100644
--- a/mach/errstring.c
+++ b/mach/errstring.c
@@ -46,8 +46,7 @@
 extern void __mach_error_map_compat (mach_error_t *);
 
 const char *
-mach_error_type( err )
-	mach_error_t		err;
+mach_error_type(mach_error_t err)
 {
 	int sub, system;
 
@@ -86,8 +85,7 @@ mach_error_string_int(mach_error_t	err,
 }
 
 const char *
-mach_error_string( err )
-	mach_error_t		err;
+mach_error_string(mach_error_t err)
 {
 	boolean_t diag;
 
